### DISCUSSION TITLE
Setting a local and a cloud Airbyte instances that can be synced easily

### DISCUSSION THREAD

**Discussion by PilarIH** [airbytehq/airbyte#34701] [Connector Questions]
Hi,
I'm trying out Airbyte to check if it could be an alternative to what we have in our infrastructure currently. But I was wondering if you guys could point me in the right direction.

We want to be able to have two instances of Airbyte. One local, as a dev environment, where we can develop without breaking anything; and another one in cloud, that would work as our production environment. The thing is that whenever we change or create something new locally, after we test it, we would like to deploy it in the prod cloud.

Is there an easy way to do this? I'm exploring Terraform to see the possibilities, but if you can offer some guidance, I would really appreciate it.

Thank you!

**Comment by sajarin** [comment]
Terraform allows you to establish infrastructure as code for the creation and management of your Airbyte connectors. 

In the Terraform setup, you'll find three main configuration files within the infra/airbyte directory:
- provider.tf: Defines the Airbyte provider, credentials, and the server URL.
- main.tf: Contains the main configuration for creating Airbyte resources.
- variables.tf: Holds various variables, including credentials.

Check out this [YouTube tutorial](https://www.youtube.com/watch?v=xVfaVrCILxE) on using Airbyte with the new Terraform provider.

[a comment by an automated account is not shown]
